build/cloudbuild_snapshot.yaml

```diff
@@ -28,7 +28,7 @@ steps:
 - name: "gcr.io/cloud-builders/gsutil"
   args: ["-h", "Content-Type:application/gzip", "cp", "kubebuilder_linux_amd64.tar.gz", "gs://kubebuilder-release/kubebuilder_master_linux_amd64.tar.gz"]
 - name: "ubuntu"
-  args: ["tar", "-zcvf", "kubebuilder_linux_arm4.tar.gz", "-C", "dist/kubebuilder_linux_arm4", "kubebuilder"]
+  args: ["tar", "-zcvf", "kubebuilder_linux_arm64.tar.gz", "-C", "dist/kubebuilder_linux_arm64", "kubebuilder"]
```

Comment:
@camilamacedo86 Correcting an old typo? Undiscovered for 2 years? Or is there a reason for this?

Reply:
It has not been running successfully so far. 
The goal with this PR is to fix it. See that it has been trigged and falling since 2020:

<img width="1018" alt="Screenshot 2022-05-08 at 23 10 09" src="https://user-images.githubusercontent.com/7708031/167317847-24e9f8c8-6238-45b9-b721-925e19b1c890.png">